src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -558,6 +558,31 @@ private function getChildren(\DOMNode $node, string $name): array
         return $children;
     }
 
+    private function getRecursiveTagAttributes(\DOMNode $node, string $missingName): array
```

Comment:
s/getRecursiveTagAttributes/getTagAttributes

Reply:
Changed in fe3bbb3a86127d13bae674f1fb7e9a5dc19470c1